app/src/main/java/se/inera/intyg/minaintyg/auth/MinaIntygUserDetailService.java

```diff
@@ -45,4 +63,17 @@ private void validatePersonId(String personId) {
       );
     }
   }
+
+  private boolean belowLoginAgeLimit(String personId) {
```

Comment:
This will work as long as we are dealing with personal identity number. If we have a coordination number we have to consider that the birth day has been modified (+60). See https://skatteverket.se/privat/folkbokforing/samordningsnummer.4.5c281c7015abecc2e201130b.html

Reply:
Added handling of coordination number